### ISSUE TITLE
Font size for Chinese words

### ISSUE THREAD

**Issue by jajazhou** [lukevp/ESC-POS-.NET#115]
i try to set the font style as :
e.SetStyles( PrintStyle.DoubleHeight | PrintStyle.DoubleWidth ),

the words include English & Chinese words, but the Chinese words cannot show as double Hight & double width, any solution?

<img width="540" alt="font size" src="https://user-images.githubusercontent.com/20143332/121834989-a530ce80-cd02-11eb-86e5-9726c9aebc66.png">

**Comment by linycyc** [comment]
hi jajazhou
I want to ask you how to print the tradition chinese word ?
I try to encode utf-8 and big5, but not working, i use virtual printer form RawBT app

Thanks~

**Comment by jajazhou** [comment]
> hi jajazhou
> I want to ask you how to print the tradition chinese word ?
> I try to encode utf-8 and big5, but not working, i use virtual printer form RawBT app
> 
> Thanks~

@linycyc 
changed this function:
        public virtual byte[] Print(string data)
        {
            // Fix OSX or Windows-style newlines
            data = data.Replace("\r\n", "\n");
            data = data.Replace("\r", "\n");

            // TODO: Sanitize...
            return data.ToCharArray().Select(x => (byte)x).ToArray();
        }
to:

        public virtual byte[] Print(string data)
        {
            // Fix OSX or Windows-style newlines
            data = data.Replace("\r\n", "\n");
            data = data.Replace("\r", "\n");

            // TODO: Sanitize...
            return Encoding.GetEncoding("BIG5").GetBytes(data);
        }

**Comment by jajazhou** [comment]
I solved this problem myself,
add the function SetFonSize to class BaseCommandEmitter,

`public virtual byte[] SetFonSize(FontStyle style) => new byte[] {  Cmd.GS, Chars.StyleMode, (byte)style };`

and the FontStyle enum as :

    [Flags]
    public enum FontStyle
    {
        None         = 0,
        DoubleHeight = 1,
        DoubleWidth  = 1 << 4 
    }

print example:
`            
```
             ..........

              e.SetFonSize( FontStyle.None), // reset to normal size

              e.SetFonSize( FontStyle.DoubleWidth), // set to double width

              e.PrintLine(somewords),

              e.SetFonSize( FontStyle.None),`  // reset to normal size

              ..........
```

**Comment by linycyc** [comment]
Hi @jajazhou 

I try to use the new function, but the RawBT app still not show traditional chinese
whether i need to use real pos printer device for my test?

```
  public virtual byte[] Print(string data)
    {
        // Fix OSX or Windows-style newlines
        data = data.Replace("\r\n", "\n");
        data = data.Replace("\r", "\n");

        // TODO: Sanitize...
        return Encoding.GetEncoding("BIG5").GetBytes(data);
    }
```

**Comment by jajazhou** [comment]
> Hi @jajazhou
> 
> I try to use the new function, but the RawBT app still not show traditional chinese
> whether i need to use real pos printer device for my test?
> 
> ```
>   public virtual byte[] Print(string data)
>     {
>         // Fix OSX or Windows-style newlines
>         data = data.Replace("\r\n", "\n");
>         data = data.Replace("\r", "\n");
> 
>         // TODO: Sanitize...
>         return Encoding.GetEncoding("BIG5").GetBytes(data);
>     }
> ```

Yes, i think so. your printer must be supported for ESC/POS standard by Epson.

**Comment by lukevp** [comment]
@jajazhou Would you be interested in contributing the FontSize changes as a PR?  Would love to get you registered as a contributor in GitHub.  If not, I can add support for this myself.  Appreciate the contribution!
